### ISSUE TITLE
Can't run acceptance tests if version of Terraform is outdated

### ISSUE THREAD

**Issue by kvrhdn** [hashicorp/terraform-plugin-sdk#559]
### SDK version

```
{
  "Path": "github.com/hashicorp/terraform-plugin-sdk/v2",
  "Version": "v2.0.1"
}
```

### Relevant provider source code

N/A

### Terraform Configuration Files

N/A

### Debug Output

```
        TF_ACC=1 go test ./...
?       github.com/kvrhdn/terraform-provider-honeycombio        [no test files]
cannot run Terraform provider tests: executable found at path /usr/local/bin/terraform is not terraform: located executable at /usr/local/bin/terraform, but output of `terraform version` was:

Your version of Terraform is out of date! The latest version
is 0.13.1. You can update by downloading from https://www.terraform.io/downloads.html
Terraform v0.13.0

FAIL    github.com/kvrhdn/terraform-provider-honeycombio/honeycombio    0.292s
?       github.com/kvrhdn/terraform-provider-honeycombio/honeycombio/internal/hashcode  [no test files]
FAIL
make: *** [testacc] Error 1
```

### Expected Behavior

I should be able to run the acceptance tests, even if my version of Terraform is not the latest.

### Actual Behavior

The acceptance tests fail to run because the output from `terraform version` is not as expected. This should output the version, but since there is a newer version available it prints a warning:

```
❯ terraform version

Your version of Terraform is out of date! The latest version
is 0.13.1. You can update by downloading from https://www.terraform.io/downloads.html
Terraform v0.13.0
```

Since my installation of terraform is managed by brew, I'd prefer not to install and update terraform manually.

### Steps to Reproduce

1. install an outdated version of terraform
1. run `TF_ACC=1 go test -v ./...` in some provider

**Comment by kvrhdn** [comment]
There was a change in the version command between terraform 0.13.0 and 0.13.1: https://github.com/hashicorp/terraform/commit/25337b7217f156b50d0de3d2cea797dbd5eb84e7
Maybe this is related?

**Comment by kvrhdn** [comment]
Closing this issue. The problem is caused by a regression in Terraform which was fixed 0.13.1. It would be pointless to try to fix this in the SDK since it's fixed now.
